storage/uri.go

```diff
@@ -111,10 +107,6 @@ func (u *uri) Fragment() string {
 	return r.Fragment
 }
 
-func (u *uri) List() ([]fyne.URI, error) {
```

Comment:
I thought we had agreed to make the existing ListableRepository List method just wrap `storage.List()`?

Reply:
Yes, but see above